chaossearch/client/operation_update_indexing_state.go

```diff
@@ -8,7 +8,8 @@ import (
 	"net/http"
 )
 
-func (client *Client) SetActive(ctx context.Context, req *SetActiveRequest) error {
+// For documentation see: https://docs.chaossearch.io/reference#bucketmodel
+func (client *Client) UpdateIndexingState(ctx context.Context, req *SetActiveRequest) error {
```

Comment:
can we align the terms used in the method (indexing state) and struct (active)? I don't care in which way, but they both should be either "active" or "indexing state".

Reply:
True, I actually planned to rename it, simply forgot. Thanks for noticing, we definitely need to make it consistent